src/visitor.rs

```diff
@@ -113,25 +148,25 @@ impl<'a, 't> NodeKind<'a, 't> {
                 match &e.data {
                     Binary(_, lhs, rhs) => {
                         ccb!(lhs.as_ref());
-                        ccb!(rhs.as_ref());
+                        ccb!(rhs.as_ref())
```

Comment:
If there are 2 following `ccb`s, shouldn't it return already if the first fails?

Reply:
Doesn't the macro just do that?